src/module/PluginsModule.ts

```diff
@@ -0,0 +1,34 @@
+import { IInstalledPlugin } from '../models/Plugins';
+import { Unauthorized } from '../response/errors';
+import { Success } from '../response/success';
+
+/**
+ * LINKURIOUS CONFIDENTIAL
+ * Copyright Linkurious SAS 2012 - 2019
+ *
+ * - Created on 2019-09-17.
+ */
+import {Module} from './Module';
+
+export class PluginsModule extends Module {
```

Comment:
You have to add the module to src/index.ts files to have the module instantiated by the rest-client and the models to index.ts for getting the types from the server.